types/stampit/stampit-tests.ts

```diff
@@ -24,22 +51,19 @@ foo.getB(); // "b"
 // Here's a mixin with public methods, and some refs:
 const membership = stampit({
     methods: {
-        members: {},
+        // members: {},
```

Comment:
Right! I remember now why this test exists. :)
Even though it's called `methods` you can put anything you want here. Including plain objects like the above `members`. So I believe the MethodsMap should be extended to allow any types:

```ts
interface MethodMap<This> {
    [s: string|Symbol]: (this: This, ...args: any[]) => any;
    [s: string|Symbol]: any;
}
```

Reply:
Can we maybe inherit `MethodMap` from `PropertyMap`?

Reply:
@koresar see my latest comment below.